app/cmd/plugin_search.go

```diff
@@ -72,19 +72,119 @@ func searchPlugins(plugins *client.AvailablePluginList, keyword string) []client
 	return result
 }
 
+func matchPluginsData(plugins []client.AvailablePlugin, pluginJclient *client.PluginManager) (result []client.CenterPlugin) {
+	if len(plugins) == 0 {
+		return
+	}
+	result = make([]client.CenterPlugin, 0)
+	jclient := &client.UpdateCenterManager{
+		JenkinsCore: client.JenkinsCore{
+			RoundTripper: pluginSearchOption.RoundTripper,
+		},
+	}
+	getCurrentJenkinsAndClient(&(jclient.JenkinsCore))
+	site, err := jclient.GetSite()
+	noSite := false
+	if err != nil && site == nil {
```

Comment:
Why not it is `err != nil || site == nil`?

Reply:
Or just make it to `noSite := (err != nil || site == nil)`

Reply:
Okay. That's good